storagenode/collector/service.go

```diff
@@ -4,30 +4,66 @@
 package collector
 
 import (
+	"context"
 	"time"
 
+	"github.com/zeebo/errs"
 	"go.uber.org/zap"
+	monkit "gopkg.in/spacemonkeygo/monkit.v2"
 
+	"storj.io/storj/internal/sync2"
 	"storj.io/storj/storagenode/pieces"
 )
 
+var (
+	mon = monkit.Package()
+
+	// Error is the default error class for piecestore monitor errors
+	Error = errs.Class("piecestore collector")
+)
+
 // Config defines parameters for storage node Collector.
 type Config struct {
-	Interval time.Duration
+	Interval time.Duration `help:"how frequently the expired pieces are cleaned" default:"0h0m5s"`
```

Comment:
I think it doesn't have to be so often

Reply:
oops meant to set back to 1hr.Missed on it and fixed

Reply:
still not fixed

Reply:
done